.github/workflows/cmake.yml

```diff
@@ -325,10 +325,15 @@ jobs:
         LSAN_OPTIONS: ${{ matrix.lsan-options || 'verbosity=1' }}
 
     - name: Upload coverage report
-      if: matrix.codecov
+      if: matrix.codecov && ( env.CODECOV_TOKEN_SECRET != '' || github.repository == 'zlib-ng/zlib-ng')
```

Comment:
There is an empty space after (

Reply:
Fixed by adding another space at the end, to keep to the style used in the file.